Sources/Database/Realm/Database.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import RealmSwift
+import Realm
+import OSLog
+
+@available(iOS 14.0, *)
```

Comment:
Just out of curiously, which part is ios14 only?

Reply:
It's the logger right?

can we just use:
```swift
if #avaialble(...) {
Logger....
}
```
for logger stuff, so that no-one confuses the whole function for being ios14+